src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportJobTask.cs

```diff
@@ -87,12 +87,20 @@ public async Task ExecuteAsync(ExportJobRecord exportJobRecord, WeakETag weakETa
                 uint currentBatchId = progress.Page;
 
                 // The first item is placeholder for continuation token so that it can be updated efficiently later.
-                var queryParameters = new Tuple<string, string>[]
+                Tuple<string, string>[] queryParameters;
+                if (!string.IsNullOrWhiteSpace(_exportJobRecord.Since))
                 {
-                    Tuple.Create(KnownQueryParameterNames.ContinuationToken, progress.ContinuationToken),
-                    Tuple.Create(KnownQueryParameterNames.Count, _exportJobConfiguration.MaximumNumberOfResourcesPerQuery.ToString(CultureInfo.InvariantCulture)),
-                    Tuple.Create(KnownQueryParameterNames.LastUpdated, $"le{_exportJobRecord.QueuedTime.ToString("o", CultureInfo.InvariantCulture)}"),
-                };
+                    queryParameters = new Tuple<string, string>[4];
+                    queryParameters[3] = Tuple.Create(KnownQueryParameterNames.LastUpdated, $"ge{_exportJobRecord.Since}");
```

Comment:
This is the most optimized implementation, but I'm thinking it might be easier to build a list for maintainability. Having array indexes that are set out of order seems like its going to trick us up at some point

Reply:
We do have UTs that cover the number and order in which the query parameters are passed to the search service. Have updated the code to use `List` anyway.